modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/info/item/provider/ObjectEntryInfoItemFieldValuesProvider.java

```diff
@@ -159,6 +160,18 @@ private List<InfoFieldValue<Object>> _getInfoFieldValues(
 			new InfoFieldValue<>(
 				ObjectEntryInfoItemFields.userNameInfoField,
 				objectEntry.getUserName()));
+		objectEntryFieldValues.add(
```

Comment:
Hey @ericthmoritsuka, could you please sort this by `ObjectEntryInfoItemFields` constants?

Feel free to message me if you have any questions :+1:

Reply:
done! thx @seltong